apps/backend/app.py

```diff
@@ -11,30 +11,25 @@
 import firebase_admin
 from firebase_admin import credentials
 from firebase_admin import firestore, storage
-from dotenv import load_dotenv
```

Comment:
Depends on #215 merging first (commenting something to block via unresolved conversation)

Reply:
Tweaking error handling